editoast/editoast_schemas/src/infra.rs

```diff
@@ -82,9 +82,30 @@ pub use track_section_source_extension::TrackSectionSourceExtension;
 pub use waypoint::Waypoint;
 
 editoast_common::schemas! {
-    track_offset::schemas(),
-    direction::schemas(),
-    track_location::schemas(),
+    applicable_directions::schemas(),
+    applicable_directions_track_range::schemas(),
+    buffer_stop::schemas(),
+    detector::schemas(),
     directional_track_range::schemas(),
+    direction::schemas(),
     electrical_profiles::schemas(),
+    electrification::schemas(),
+    endpoint::schemas(),
+    loading_gauge_limit::schemas(),
+    neutral_section::schemas(),
+    operational_point::schemas(),
+    route::schemas(),
+    side::schemas(),
+    signal::schemas(),
+    sign::schemas(),
+    speed_section::schemas(),
+    switch::schemas(),
+    switch_type::schemas(),
+    track_endpoint::schemas(),
+    track_location::schemas(),
+    track_offset::schemas(),
+    track_section_extensions::schemas(),
+    track_section::schemas(),
+    track_section_sncf_extension::schemas(),
+    track_section_source_extension::schemas(),
```

Comment:
We should inline structs that will only be referenced once. Like `track_section_extensions`, `track_section_sncf_extension`, and so on...

Reply:
Done.